### DISCUSSION TITLE
Is the edk2toolext released in the form of .whl file?

### DISCUSSION THREAD

**Discussion by jpshivakavi** [tianocore/edk2-pytool-extensions#869] [Q&A] [answered]
- [ ]

**Comment by Javagedes** [comment, marked as answer]
Hello, we upload to pypi as .whl and as a .tar.gz

See https://pypi.org/project/edk2-pytool-extensions/#files

Thanks,
Joey

**Reply by jpshivakavi** [reply]
Thanks for your reply. Initially I tried installing with edk2toolext package name through pip install. I didn't work, so I thought of checking with the project team. Thanks for the pypi package link.
